src/com/ebcitron/piggybank/Dollar.java

```diff
@@ -0,0 +1,15 @@
+package com.ebcitron.piggybank;
+
+public class Dollar extends Coin {
+
+    public double value = 1;
+
+    public Dollar(double quantity) {
+        super(quantity);
+    }
+
+
+    public double getTotal() {
+        return value * this.getQuantity();
```

Comment:
Just wanted to point out that the `this` isn't necessary here, but it's totally cool to have it!